develop-guide/writeparser/writeparser.rst

```diff
@@ -827,24 +846,26 @@ Using ``parsergen`` to create a parser class is particularly useful when you don
 
 Now that you've finished writing your parser, the time has come to test it!
 
-There are currently two mechanisms to create tests as the testing strategy is
-in a state of transition. The reasoning for the new testing type is to avoid
-merge conflicts, removing duplicate boilerplate code, ensuring tests are built,
-and making the process overall easier. However, in the meantime unfortunately
-a user may create different tests depending on the OS of the parser that
-they wrote.
+There are currently two types of tests, those being Folder Based and Unittest Based.
+The testing strategy is currently in a state of transition.
+Moving to folder based testing helps us avoid merge conflicts and duplicate boilerplate code
+while ensuring tests are faster and easier create and use.
+When creating tests for a new parser do so in the Folder Based style.
 
-:ref:`Folder based testing <folder_based_testing>` is currently supported for:
+:ref:`Folder based testing <folder_based_testing>` is currently completed for:
 
   * ASA
   * IOS
   * IOSXE
+  * IOSXR
+  * JUNOS
+  * NXOS
```

Comment:
as you can see here, NXOS is partially done. not fully done, I believe...
https://github.com/CiscoTestAutomation/genieparser/tree/master/src/genie/libs/parser/nxos/tests